src/SprykerEco/Yves/Adyen/Controller/NotificationController.php

```diff
@@ -16,9 +16,8 @@
  */
 class NotificationController extends AbstractController
 {
-    protected const HEADER_CONTENT_TYPE_KEY = 'Content-Type';
-    protected const HEADER_CONTENT_TYPE_VALUE = 'application/json';
-    protected const NOTIFICATION_ACCEPTED_RESPONSE = ['notificationResponse' => '[accepted]'];
+    protected const NOTIFICATION_ACCEPTED_RESPONSE_HEADER = ['Content-Type' => 'application/json'];
```

Comment:
Call the const so that its name is meaningful. Like `NOTIFICATION_RESPONSE_HEADER_CONTENT_TYPE`